wiki/FitNesseRoot/HsacExamples/SlimTests/BrowserTests/Angular2CalculatorTest.wiki

```diff
@@ -0,0 +1,28 @@
+Sample test on [[Angular][https://angular.io/]] application. 
+
+!note '''Please note:''' The application used in this test does NOT seem to work with !-PhantomJs-!. It does work when executed with Chrome or Firefox.
+
+!note Due to the nature of [[Plunker][https://plnkr.co/]] which hosts the application in an iframe, the page must first be opened with !-BrowserTest-!, switch to the embedded iframe and wait for the application to load.
+
+
+|table template|add                      |
+|enter         |@{first} |as |id=first   |
+|enter         |@{second}|as |id=second  |
+|select        |+        |for|id=operator|
+|click         |Go!                      |
+|$result=      |value of |id=latest      |
+
+|script                |Browser Test                                |
+|seconds before timeout|15                                          |
+|set browser size to maximum                                        |
+|open                  |https://embed.plnkr.co/2GtNBG6QrG8AL0l9rxjc/|
+|switch to frame       |css=.plunker-space-iframe                   |
+|switch to frame       |id=preview                                  |
+|wait for visible      |id=calculator-container                     |
+
+|script|Ng2 Browser Test|
+
```

Comment:
When the service was working I thought the test looked a bit better when I re-wrote the scripts to:
```
|script                |Browser Test                                |
|seconds before timeout|25                                          |
|open                  |https://embed.plnkr.co/2GtNBG6QrG8AL0l9rxjc/|
|switch to frame       |css=.plunker-space-iframe                   |
|switch to frame       |id=preview                                  |

|script          |Ng2 Browser Test|
|wait for visible|Super Calculator|

```